modules/secure-preview/rest/secure-preview.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * class EditStatus
```

Comment:
Should change this name and the description below

Reply:
Changed in https://github.com/Automattic/vip-workflow-plugin/pull/19/commits/283371c204bacd6e1cf3f6fc2c4d301ed4416f56.